Makefile

```diff
@@ -47,15 +48,25 @@ else
 endif
 
 flake8:
-	flake8
+	flake8 --ignore=E203,E731,E402,W503,W504
 
 test: clean
 	pytest --random-order
 
 coverage: clean
 	pytest --random-order --cov-config .coveragerc --cov-report term-missing --cov=mu tests/
 
-check: clean flake8 coverage
+tidy: clean
```

Comment:
This Makefile rule ends up needing to be "duplicated" in Windows' `make.py` equivalent... I'm wondering if there could be a way of avoiding such repetition, for increased maintainability. :)

Maybe a [`pyproject.toml`](https://github.com/psf/black#pyprojecttoml) file could be created with a `[tool.black]` section specifying the correct `line-length` and `exclude` values such that "just running" `black` does what we want. That might make it DRY and work the same way on any platform, Windows or UNIX-line. :)

Reply:
It would have been nice if flake8 supported `pyproject.toml` as well (https://gitlab.com/pycqa/flake8/issues/428), that way we could have replaced the current `setup.cfg`.
I guess we'll need to add yet another file :/ (can't wait until `pyproject.toml` takes over the world!).

Reply:
@carlosperate, I'm with you. At least WRT `flake8` getting its configuration from `pyproject.toml`... But that issue and associated MR (PR in GitLab's words) seem to be somewhat on hold. The discussion there is interesting though. :)